server/base/backend.js

```diff
@@ -140,3 +141,5 @@ export const Chatpal = {
 	service: {},
 	Backend: new ChatpalBackend(SystemLogger)
```

Comment:
Earlier on, the `init()` was executed inside the constructor.
Since `Chatpal` was exported (and thus not inside the `startup()`), a failure of the initialization failed the actual startup.